### PR TITLE
Doc fix: formats and typo.

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

**Comment by QuanluZhang** [comment]
suggest to change back to CRLF

**Comment by PurityFan** [comment]
Great job! 
But I am a little worried, because this pr and the branch of dev-doc modify the same document, such as "Overview". And this two modification seems to be difficult to merge due to lots of **conflict**, I am wondering if we will eventually have a big trouble when merging?

File: .gitignore [hunk 1]
```diff
@@ -67,3 +67,5 @@ typings/
 __pycache__
 build
 *.egg-info
+
```
Comment (leelaylay) [inline .gitignore]: I think that we can remove `.vscode` in gitignore.

File: docs/GetStarted.md [hunk 2]
```diff
@@ -2,27 +2,35 @@
...
+  git
+  wget
+  ```
+
+  python pip should also be correctly installed. You could use "python3 -m pip -v" to check in Linux.
 
-    python pip should also be correctly installed. You could use "python3 -m pip -v" to check in Linux.
-    
-    * Note: we don't support virtual environment in current releases.
+  * Note: we don't support virtual environment in current releases.
```
Comment (leelaylay) [inline docs/GetStarted.md]: It seems that virtual environment like conda is supported if we install it in the right way.

File: .gitignore (hunk 1 above)
Comment (squirrelsc) [reply .gitignore]: .vscode is for vs code local settings. It's better to keep it local, instead of sharing.

File: docs/GetStarted.md (hunk 2 above)
Comment (squirrelsc) [reply docs/GetStarted.md]: There is no change on original content, I just changed indent of those lines. We can review content again, and get it correctly.

**Pull request merged**